numpy/core/src/umath/ufunc_object.c

```diff
@@ -5572,6 +5603,8 @@ ufunc_get_identity(PyUFuncObject *ufunc)
         return PyInt_FromLong(1);
     case PyUFunc_Zero:
         return PyInt_FromLong(0);
+    case PyUFunc_MinusOne:
```

Comment:
They don't matter ;) It is a getter function for a property of the ufunc and are only there to make the number visible to python users.